app/config/config.yml

```diff
@@ -199,3 +199,42 @@ scheb_two_factor:
 kphoen_rulerz:
     executors:
         doctrine: true
+
+fos_elastica:
+    clients:
+        default: { host: localhost, port: 9200 }
+    serializer:
+        callback_class: FOS\ElasticaBundle\Serializer\Callback
+        serializer: serializer
+    indexes:
+        wallabag:
+            client: default
+            settings:
+                index:
+                    analysis:
+                        analyzer:
+                            custom_french_analyzer:
+                                type: custom
+                                tokenizer: letter
+                                filter: ["asciifolding", "lowercase", "french_stem", "stop_fr"]
```

Comment:
There is no way you will be able to get a good analysis if you don't know the language of the content, so my best guess would be to build a triGram analyzer, with html_strip char filter etc... and use multi-fields.

Each "searchable field" could be mapped:

- with a triGram analyzer (to collect a lot of contents)
- with standard analyzer (to improve the pertinence)

It will not be perfect :disappointed: and will need some tuning when it's setup (there are so many ways to do analysis, you have to found one that will fit your contents and how you want to search them).